routes/api/user/account.js

```diff
@@ -5,37 +5,37 @@ const UserCollections = require('models/User')
 const log = require('common/log')
 
 module.exports = {
-	get: function(req, res){
-		if(!req.session.passport.user){
-			return res.status(500).send()
-		}
-		let conditions = {
-			_id: req.session.passport.user
-		}
-		let props = req.query.props
-		UserCollections.findOne(conditions, function(err, user){
-			if(err){
-				return log.info(err)
-			}
-			let ret = {}
-			ret[props] = user[props]
-			res.send(ret)
-		})
-	},
+  get: function(req, res){
+    if(!req.session.passport.user){
+      return res.status(500).send()
+    }
+    let conditions = {
+      _id: req.session.passport.user
+    }
+    let props = req.query.props
+    UserCollections.findOne(conditions, function(err, user){
+      if(err){
+        return log.info(err)
+      }
+      let ret = {}
+      ret[props] = user[props]
+      res.send(ret)
+    })
+  },
 
-	put: function(req, res){
-		if(!req.session.passport.user){
-			return res.status(500).send()
-		}
-		let user = User(req.session.passport.user)
-		let property = req.body.property
-		let data = req.body.data
+  put: function(req, res){
+    if(!req.session.passport.user){
+      return res.status(500).send()
+    }
+    let user = User(req.session.passport.user)
```

Comment:
__[jshint]__
*  'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).

*  Missing 'new' prefix when invoking a constructor.

*  Missing semicolon.